stellar-contract-env-host/src/test.rs

```diff
@@ -1173,3 +1168,197 @@ fn binary_xdr_roundtrip() -> Result<(), HostError> {
     assert_eq!(host.obj_cmp(obj.into(), obj_back.into())?, 0);
     Ok(())
 }
+
+#[test]
+fn bigint_tests() -> Result<(), HostError> {
+    let host = Host::default();
+    let a: u64 = 2374340;
+    let b: i64 = -438730;
+    // let bi_0: BigInt = 0_u32.into();
```

Comment:
Nitpick: why are these commented out?

Reply:
These are convenience code to generate the result reference. I think it's convenient to have in case one may change the value `a` and `b` and need to regenerate the references.

Reply:
I don't understand. What result reference? Whatever the result reference is, why would you need to regenerate?

Even if this is convenience code for some purpose that I don't yet understand, there's no indication that it is convenience code for whatever that purpose is. As such, it's very unlikely that a maintainer would be able to make use of it as currently presented.

Reply:
Remove them.